dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -148,6 +149,7 @@ export const renderElement = ({
 	forceDropCap,
 	isTimeline = false,
 	totalElements = 0,
+	isListElement,
```

Comment:
`isTimeline` defaults to `false`, should we be consistent here?

Reply:
Yep seems like a good plan - i'll update